docs/wiki/Getting started - Scenario 2 Onboard module library and CI environment.md

```diff
@@ -6,7 +6,6 @@ This requires several steps:
 1. [Fork/clone the repository into your DevOps environment](#2-forkclone-the-repository-into-your-devops-environment)
 1. [Configure the CI environment](#3-configure-the-ci-environment)
 1. [Deploy dependencies](#4-deploy-dependencies)
```

Comment:
Should we explicity call it "manual" dependencies?

Reply:
done